tests/test_fsclient.py

```diff
@@ -134,6 +133,22 @@ def test_insert_one(make_db, rc, input, result):
     assert list(client.dbs[dbname][collname].values())[-1] == result
```

Comment:
per your question (which I don't see here for some reason) this may be left over from Regolith where the process is to load the entire database into memory (as client.dbs[dbname][collname]).  It is then put back into the database at the end with a `db.dump()` or sthg like that.  But I think we don't want to do that here as we will have 100,000 entries in the db.  so we want `insert_one` to insert it all the way into the database backend.  In the filesystem case this will be a text file on disc containing yml or json.  in the mongo case it will be a remote mongodb and for mpcontribs it will be in mpcontribs.

Reply:
Sorry, quick questions here. So it's a bad practice to load client.dbs[dbname][collname] entirely using the above.> What if we implement find_one first and use it to test insert_one? For example: `assert client.find_one(inserted_cif_id) ==  inserted_cif`?

Reply:
I think we could, but I am not convinced that it is the best way.  We should presumably also test that nothing else in the file was corrupted or overwritten or sthg.

Also, as I mentioned before, we are not inserting cifs per se.  I think insert_one should be able to insert any valid json.  I think you agree, but by putting cif in the name it makes the test harder to read so I suggest changing `cif` to `json` in the name.

Reply:
Thank you! Also another quick question, by inserting all the way into the database backend you mean overriding the yaml file directly instead of using client.load_db and client.dump_db? That means we have to change our insert_one function to directly access the yaml file?

Reply:
yes, I think that is right.  Regolith assumed small databases that could be stored in memory, then operations on the items could be done with do loops and list comprehensions, but with large databases we want to use the powerful database capabilities directly, so we want kind of CRUD capabilities.    We may probably want the fs to be json format not yml, but otherwise, yes, I think we want to insert directly.